### DISCUSSION TITLE
New operation status needs adaptation?

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#52] [Shipped - Live on production]
I like the new Active/Active (limited use)/Active (out of operation) statuses. One remark from my side is that I would expect the status quick filter Active would include both limited use and out of operation (as those are still active). Similar to how Active (Incomplete) is also included in Active status. Unless Active actually means Fully Operational?

**Comment by YvesR** [comment]
We do provide one filter for each status. So yes, the previous existed `active` is the one used for fully functional now.

**Comment by BioMedGPH** [comment]
In my opinion Active and Retired are 'Administrative' statuses, and 
Fully functional/limited use/out of operation are 'Operational' 
statuses. So it is being mixed up like apples and bananas, in my 
opinion. When I'm reviewing the list I would like to see all devices, 
regardless of operational status (but be able to filter between various 
operational statuses). But I only want active assests, not retired 
assets (or chose Show All to see both active and retired assets).

..................
Bio-Medical Engineering department
INF Nepal, Green Pastures Hospital
Hariyokharka,Pokhara

**Reply by YvesR** [reply]
I understand your point, but it is mixed up on purpose. But I agree that we should clarify more the status. I think we can rename "Active" in selection as "Fully functional", as all 3 status results in an "active" device.

**Comment by kabulNoorBiomed** [comment]
I would like there to be an option for a device which is fully functional, but not active, even if the device is not retired. For example, we have devices in our inventory which are functional, but they are not in used because they are surplus, staff are not trained in how to use them, or they are new and not yet installed. These could all be in the same category of inactive but functional. It would be helpful for us to know which machines we have surplus of, especially as the machines we are using come to end of life and need replacing.

**Reply by YvesR** [reply]
We do have `Operational`, `Limited use`, `Out of order` and `Retired` as options at the moment.
As filter we combine `Active (all)` and other and as special case `Retired`.

For your purpose I think we should have another status `inoperative` that indicates it is available, functional, we even do PPM, but they are not in use for any reason. If we choose this we should enter a `reason`as description that is shown in the grid page, landing page and inside the form. Description can be any text you like to use, which you can also use to filter in the grid.
This would be my generic approach here that should cover most cases and easy to implement.

@BioMedGPH  @kabulNoorBiomed 
What you guys think?

**Reply by BioMedGPH** [reply]
fully functional/limited use/out of order: those are functional/technical statuses, it is not the hospitals decision, but follows from the state of the machine.
active use/non-active/retired are administrative statuses, it is the hospitals choice to put them in that status. Any of these statuses can be combined with functional/limited use/out of order.
So just keep them as separate columns/variables that can independently be filtered to create any combination you like.

**Comment by BioMedGPH** [comment]
I support this; there might even be an older request from me for this. The functional status (fully functional, limited use, out of operation) and operational status (active use, non-active, retired) are separate entities and should be possible to choose these independently. Also there is already Device Condition (a free text field, but maybe predefined fields that can be chosen are easier to use and give more uniform data), that is a 3rd descriptor, mainly to assess it's overall condition (normal, below normal, replacement near, to be replaced, etc). These 3 descriptors are all valuable, but are generally independent and should be treated as such.

**Reply by BioMedGPH** [reply]
in addition: operational status (active use, non-active, retired) has impact on maintenance planning, as retired requires no maintenance, active requires full maintenance, and non-active might have a reduced maintenance, or no maintenance at all.

**Comment by BioMedGPH** [comment]
Maybe a discussion about the labels of these 3 categories and some discussion about the different values for levels within these 3 categories are required to make clear distinctions. Happy to have a chat on Zoom etc.
